src/lib/idleTimeout.js

```diff
@@ -132,12 +142,14 @@ export const handleIdleTimeout = async (idleTimeoutConfig) => {
 
 	await store.setState({
 		idleTimeout: {
-			idleWarningTimer: warningTimer,
-			idleTimeoutTimer: timeoutTimer,
 			idleTimeoutMessage,
 			idleTimeoutWarningTime,
 			idleTimeoutTimeoutTime,
 			idleTimeoutRunning: idleTimeoutAction === 'start',
 		},
+		idleTimeoutTimer: {
```

Comment:
Can we rename this one to `idleTimeoutTimers`? Other things looks good to me 🎉

Reply:
thanks @Shailesh351 .........  @bhardwajaditya if you can make this small change real quick that will allow us to include this PR in the new release.  thanks.

Reply:
@ear-dev @Shailesh351 Done